examples/image_classifier/mnist/README.md

```diff
@@ -34,6 +34,42 @@ Run the commands given in following steps from the parent directory of the root
     curl http://127.0.0.1:8080/predictions/mnist -T examples/image_classifier/mnist/test_data/0.png
     ```
 
+# Profiling inference output
+
+To enable pytorch profiler, set the following environment variable.
+
+```
+export ENABLE_TORCH_PROFILER=true
+```
+
+Note: Ensure`enable_envvars_config` is set to `true` in torchserve config
+
+Run the predictions again
+
+```
+curl http://127.0.0.1:8080/predictions/mnist -T examples/image_classifier/mnist/test_data/0.png
+```
+
+The profiler information is printed in the torchserve logs / console 
+
+![Profiler Stats](screenshots/mnist_profiler_stats.png)
+
+By default the pytorch profiler trace files are generated under "/tmp/pytorch_profiler/<model_name>" directory.
+
+The path can be overridden by setting `on_trace_ready` parameter in `profiler_args` - [Example here](../../../test/pytest/profiler_utils/resnet_profiler_override.py)
+
+And the trace files can be loaded in tensorboard using torch-tb-profiler. Check the following link for more information - https://github.com/pytorch/kineto/tree/main/tb_plugin 
+
+Install torch-tb-profiler and run the following command to view the results in UI
+
+```
+tensorboard --logdir /tmp/pytorch_profiler/mnist/
```

Comment:
@shrinath-suresh in the handler below I think, "on_trace_ready" to override the path is missing.

Reply:
on_trace_ready is not a mandatory argument. If user provides the argument, the trace files will get generated into the location. if the user doesn't provide the argument, the trace files get generated inside "/tmp/pytorch_profiler/<model_name>" folder. Its handled [here](https://github.com/shrinath-suresh/serve/blob/19a521189e574f07cd465cde8019b59ee73d3c13/ts/torch_handler/base_handler.py#L245)

For demonstration, we can add on_trace_ready in handler file. Again, we cannot set the `on_trace_ready` to `os.getcwd()` as it will be a temp directory in run time. May be we can try use home directory or something .. Your thoughts ?